tmt/steps/discover/fmf.py

```diff
@@ -82,6 +185,31 @@ def wake(self):
             if value:
                 self.data[option] = value
 
+    def check_requires(self, all_requires):
+        """
+        Check requires for possible beakerlib libraries
```

Comment:
requires and recommends, or dependencies to say it shortly.

Reply:
Updated.